composer.json

```diff
@@ -16,11 +16,11 @@
     },
     "require": {
         "php": ">=7.1",
-        "cakephp/cakephp": "^3.6",
-        "league/flysystem": "^1.0"
+        "cakephp/cakephp": "dev-master",
+        "league/flysystem": "^1.0",
+        "phpunit/phpunit": "^8.0"
```

Comment:
ideally `"phpunit/phpunit` should be in `require-dev` section

Reply:
I fixed that in a newer commit, was faster than doing a review and waiting until the stuff is in there :)

Reply:
Awesome thanks